src/Commands/ExportPostmanCommand.php

```diff
@@ -93,6 +108,24 @@ public function handle(): void
                     continue;
                 }
 
+                if ($this->config['include_doccomments']) {
+                    try {
+                        $docComment = $reflectionMethod->getDocComment();
+                        $tokens = new TokenIterator($this->lexer->tokenize($docComment));
+                        $phpDocNode = $this->phpDocParser->parse($tokens);
+
+                        foreach ($phpDocNode->children as $child) {
+                            if ($child instanceof PhpDocTextNode) {
+                                $requestDescription .= ' ' . $child->text;
+                            }
+                        }
+                        $requestDescription = Str::squish($requestDescription);
+                    } catch (\Exception $e) {
+                        $requestDescription = 'Error at parsing phpdoc at ' . $reflectionMethod->class . '::' . $reflectionMethod->name;
```

Comment:
I don't think it's helpful for the postman collection to contain errors about parsing the doc blocks? Can we just show a friendly error in the console and move on?